### PR TITLE
Add limits to Thanos pods - see OSD-6250

### PR DESCRIPTION
**Pull request by csheremeta**
https://issues.redhat.com/browse/OSD-6250
https://issues.redhat.com/browse/OHSS-2530
https://bugzilla.redhat.com/show_bug.cgi?id=1906496
https://access.redhat.com/solutions/5685771

### CODE REVIEW

File: deploy/cluster-monitoring-config/cluster-monitoring-config.yaml [hunk 1]
```diff
@@ -70,3 +70,11 @@ data:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
           operator: Exists
+    thanosQuerier:
```
Comment (jewzaam) [inline deploy/cluster-monitoring-config/cluster-monitoring-config.yaml:73]:
```suggestion
    # https://access.redhat.com/solutions/5685771
    # https://bugzilla.redhat.com/show_bug.cgi?id=1906496
    thanosQuerier:
```

**Review by jewzaam — CHANGES_REQUESTED** [review]
I think comments in the yaml will work, will help us by nothing that there's a reason this is in there and it can maybe go away eventually.

File: deploy/cluster-monitoring-config/cluster-monitoring-config.yaml (hunk 1 above)
Comment (csheremeta) [reply deploy/cluster-monitoring-config/cluster-monitoring-config.yaml:75]: done :)

**Review by fahlmant — APPROVED** [review]
/lgtm

**Pull request merged**